src/View/Helper/MigrationHelper.php

```diff
@@ -103,6 +103,10 @@ public function columnMethod($action)
             return 'removeColumn';
         }
 
+        if ($action === 'alter_field') {
+            return 'changeColumn';
+        }
+
```

Comment:
just a question : should not apply this to `indexMethod()` too ?

Reply:
Thank you for your quick response. I didn't check about indexMethod. Logically user can create index through alter. So, it should. I will check and response shortly. 
Thank you very much

Reply:
Just checked the functionality of `indexMethod`. Here have two observations. 
Someone can set index while creating and later just change the field type or any other attributes, keeping the indexed. 

Someone can create the index again. So, I think, for modifying index, should use another function to drop index. What do you say? This is just my thoughts.